index.js

```diff
@@ -41,6 +43,21 @@ async function run() {
       console.log(`   - ${key} ${chalk.dim(`(used in ${generateFileList(files)})`)}`);
     }
   }
+  console.log();
+
+  console.log(`${step(4)} ⚙️   Checking for missing translations...`);
+  let missingTranslations = findDifferenceInTranslations(usedTranslationKeys, existingTranslationKeys, whitelist);
```

Comment:
can you move this step above the `unusedTranslations` reporting? I would like to have all the processing done before we start reporting the final results.

Reply:
Yep, good idea. I can do so in the next hour, thanks.